pmpro-member-directory.php

```diff
@@ -158,3 +158,53 @@ function pmpromd_plugin_row_meta($links, $file) {
 	return $links;
 }
 add_filter('plugin_row_meta', 'pmpromd_plugin_row_meta', 10, 2);
+
+/**
+ * Filters through the fields we're expecting to show and make sure the user has the required level
+ * 
+ * @param  array $profile_fields The fields we want to display on the page
+ * @param  object $pu             The current user object
+ * @return array Returns the fields we want to show
```

Comment:
```suggestion
 * Filter the fields we are expecting to show and make sure the user has the required level.
 *
 * @since TBD
 * 
 * @param array  $profile_fields The list of fields we want to display on the page.
 * @param object $pu             The current user object.
 *
 * @return array The list of fields we want to display on the page.
```